src/component/mxgraph/shape/render/BpmnCanvas.ts

```diff
@@ -19,8 +19,10 @@ import type { IconConfiguration, IconStyleConfiguration, ShapeConfiguration, Siz
 import type { mxAbstractCanvas2D } from 'mxgraph';
 
 /**
+ * **WARN**: You may use it to customize the BPMN Theme as suggested in the examples. But be aware that the way the default BPMN theme can be modified is subject to change.
```

Comment:
The links were previously broken, this is what was generated with typedoc@0.22.17

![image](https://user-images.githubusercontent.com/27200110/177130735-0a3e9e8d-545f-46b8-8845-a56b56a63729.png)

With the changes proposed here, we have the following

![image](https://user-images.githubusercontent.com/27200110/177130894-b7bb6bc1-94a9-42bf-ae40-7dc007d6d51f.png)